lib/routes.jsx

```diff
@@ -46,11 +46,33 @@ var pages = {
  */
 var redirects = {
   'clubs/curriculum': 'activities/web-lit-basics',
-  'teach-like-mozilla/web-literacy': 'activities/web-literacy'
+  'teach-like-mozilla/web-literacy': 'activities/web-literacy',
```

Comment:
might as well point this straight to `web-literacy` too now, no need for a double redirect